{"frames":[{"func":"read.csv(paste0(getwd(), \"/\", rstanmodelPath, \"/models/\", linear_model_path, \n    \"/rlt/m_predErr_\", linear_model, \".csv\"))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"read.table(file = file, header = header, sep = sep, quote = quote, \n    dec = dec, fill = fill, comment.char = comment.char, ...)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"file(file, \"rt\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in file(file, \"rt\") : cannot open the connection\n"}